### DISCUSSION TITLE
Registration is Closed eventhough it's enabled

### DISCUSSION THREAD

**Discussion by wisefella** [simple-login/app#1400] [Self-hosting question]
This is a new self-hosted install.  

After creating the first user, I disabled the registration & onboarding (R&O).  Then later I re-enabled the R&O and restart the sl-app, but I continue to get the error on the web page stating that the registration is closed.  I also restarted all the containers but to no avail.

**Comment by nguyenkims** [comment]
@wisefella how did you disable/enable the registration? Please note that in the SL config file, the value of a boolean option isn't determined by its value but rather whether it's present or not in the config file.

**Reply by wisefella** [reply]
Thanks @nguyenkims 

I did end up figuring that out.  However, I'm still stuck at not being able to do the initial login as I'm not able to get the activation email.  I've spent a few hours trying to figure that but to no avail.  Here's the basic setup I have:  
- Running SL on a VM
- Port forwarding 25, 80/443
=====
Running swaks  generates no log entries in any of the containers (sl-app, sl-email, sl-db, sl-job-runner).
On the terminal running swaks with two email ids generates:
swaks --to myid@outlook.com  => Shows === Trying outlook-com.olc.protection.outlook.com:25...
swaks --to myid@gmail.com => Shows === Trying gmail-smtp-in.l.google.com:25...
=====
Does this seem like a outbound SMTP connectivity issue on port 25?  Why is the sl app connecting on port 25 for outlook & gmail.  I thought these providers use 587/submission
=====
Before attempting a login, I do see a 
`ERROR:  relation "domain" does not exist at character 15" in sl-db container.`
`STATEMENT:  select * from domain;`
=====
Rest of the container logs look normal:
`sl-email: email_handler.py:1653 - main() -  - Start mail controller 0.0.0.0 20381`
`sl-app:   /code/server.py:444 - after_request() -10.0.0.1 GET /auth/login ImmutableMultiDict([]) 200`
sl-job-runner:
` "job_runner.py:114" - <module>() -  - Take job <Job 4 onboarding-1 {'user_id': 2}>`
` "job_runner.py:114" - <module>() -  - Take job <Job 2 onboarding-2 {'user_id': 1}>`
` "job_runner.py:114" - <module>() -  - Take job <Job 5 onboarding-2 {'user_id': 2}>`
` "job_runner.py:114" - <module>() -  - Take job <Job 3 onboarding-4 {'user_id': 1}>`
=====
Attempting to login, generates the following log entry in sl-app:
` "/code/server.py:444" - after_request() -  - 10.0.0.1 POST /auth/login ImmutableMultiDict([]) 200`
========

**Reply by nguyenkims** [reply]
The `ERROR: relation "domain" does not exist at character 15" in sl-db container.` error is quite strange as there's no table named "domain" in SimpleLogin. I'd double check the set up process, especially the db migration step.
